src/app/global-vars.service.ts

```diff
@@ -137,7 +124,7 @@ export class GlobalVarsService {
   followFeedPosts = [];
   hotFeedPosts = [];
   tagFeedPosts = [];
-  messageResponse = null;
+  decryptedMessages: DecryptedMessageEntryResponse[] | null = null;
```

Comment:
not sure if it's needed

Reply:
I'm assuming this is to handle the new message methodology, but. we can dig deeper together